test/test_subword.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from torchtext import data
+from torchtext.datasets import TREC
+
+
+class TestSubword(unittest.TestCase):
+    def test_subword(self):
+        TEXT = data.SubwordField()
+        LABEL = data.Field(sequential=False)
+        RAW = data.Field(sequential=False, use_vocab=False)
+        raw, = TREC.splits(RAW, LABEL, train='TREC_10.label',
+                           validation=None, test=None)
```

Comment:
There is no validation set available for TREC, but this would work: 

`raw, = TREC.splits(RAW, LABEL, train=TREC.test_filename, test=None)`